src/main/java/com/mathworks/ci/RunMatlabTestsStep.java

```diff
@@ -142,6 +153,11 @@ private String getInputArgs() {
         args.put("SimulinkTestResults", getTestResultsSimulinkTest());
         args.put("CoberturaCodeCoverage", getCodeCoverageCobertura());
         args.put("CoberturaModelCoverage", getModelCoverageCobertura());
+        if(getSourceFolder() != null && getSourceFolder().size() != 0){
```

Comment:
Using isEmpty is more idiomatic that size != 0

Reply:
Changed to isEmpty